src/convert_array_type_test.cpp

```diff
@@ -0,0 +1,81 @@
+#include <fstream>
+#include <iostream>
+
+#include "Omega_h_vtk.hpp"
+#include "Omega_h_file.hpp"
+
+#ifdef OMEGA_H_USE_ADIOS2
+#include <Omega_h_adios2.hpp>
+#endif
+
+using namespace Omega_h;
+
+void check_SymmetricSquareMatrix_tag(const Tag<Real>* tag) {
+  OMEGA_H_CHECK(tag->array_type() == ArrayType::SymmetricSquareMatrix);
+  OMEGA_H_CHECK(tag->ncomps() == 3);
```

Comment:
As pointed out by Fuad in https://github.com/SCOREC/omega_h/issues/180, we should use something here that won't be compiled away when a release build is used.   We can address this in a another PR.